samples: nrf9160: lwm2m_client: Generate timestamp for location (#8234)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    * Bootstrap is not TLV only anymore.
      With v1.1, preferred content format is sent in the bootstrap request.
      SenML CBOR takes precedence over SenML JSON and OMA TLV, when enabled.

>     * Generate the timestamp of LwM2M location object when we get location fix.

Reviewer: ```suggestion
    * Generation of the timestamp of LwM2M Location object on obtaining location fix.
```
